modules/@apostrophecms/area/ui/apos/components/AposAreaMenu.vue

```diff
@@ -102,7 +102,7 @@ export default {
       type: Boolean
     }
   },
-  emits: [ 'menu-close', 'menu-open', 'insert' ],
```

Comment:
`AposAreaWidget` is still listening for `@menu-close`

Reply:
re implemented